tests/func/test_checkout.py

```diff
@@ -101,7 +103,8 @@ def test(self):
         with open(cache, "w+") as fobj:
             fobj.write("1")
 
-        self.dvc.checkout(force=True)
+        with pytest.raises(DvcException):
```

Comment:
Should it be
```suggestion
        with pytest.raises(CheckoutError):
```
here too?